### PR TITLE
Add example of using Dask to parallelize to docs 

### PR DESCRIPTION
**Pull request by WillKoehrsen**
Update of the documentation to point to the [Featuretools on Dask notebook](https://github.com/Featuretools/Automated-Manual-Comparison/blob/master/Loan%20Repayment/notebooks/Featuretools%20on%20Dask.ipynb). Also added in section to parallel about partitioning data and running on multiple cores.

### CODE REVIEW

File: docs/source/guides/parallel.rst [hunk 1]
```diff
@@ -50,3 +49,7 @@ The dashboard requires an additional python package, bokeh, to work. Once bokeh
                                      n_jobs=2,
                                      dask_kwargs={'diagnostics_port': 8787}
                                      verbose=True)
+                                     
+Parallel Computation by Partioning Data
+-------------------------------
+As an alternative to Featuretool's parallelization, the data can be partitioned and run on multiple cores or a cluster using Dask or PySpark. For more information on partitioning the data and using Dask, see :doc:`/guides/performance`. An example of this approach can be seen in the `Featuretools on Dask notebook <https://github.com/Featuretools/Automated-Manual-Comparison/blob/master/Loan%20Repayment/notebooks/Featuretools%20on%20Dask.ipynb>`_. Dask allows Featuretools to scale to multiple cores on a single machine or multiple machines on a cluster.
```
Comment (kmax12) [inline docs/source/guides/parallel.rst]: just link to the other place in docs, no need to have link twice. that also means we once have to keep once place up to date.

File: docs/source/guides/performance.rst [hunk 2]
```diff
@@ -84,6 +84,8 @@ When an entire dataset is not required to calculate the features for a given set
 
 An example of this approach can be seen in the `Predict Next Purchase demo notebook <https://github.com/featuretools/predict_next_purchase>`_. In this example, we partition data by customer and only load a fixed number of customers into memory at any given time. We implement this easily using `Dask <https://dask.pydata.org/>`_, which could also be used to scale the computation to a cluster of computers. A framework like `Spark <https://spark.apache.org/>`_ could be used similarly.
 
+An additional example of partitioning data to distribute on multiple cores or a cluster using Dask can be seen in the `Featuretools on Dask notebook <https://github.com/Featuretools/Automated-Manual-Comparison/blob/master/Loan%20Repayment/notebooks/Featuretools%20on%20Dask.ipynb>`_. Dask allows us to easily scale to multiple cores on a single computer or multiple machines on a cluster.
```
Comment (kmax12) [inline docs/source/guides/performance.rst]: let's also mention there is a blog post about it and link to the TDS blog as well

Comment (WillKoehrsen) [reply docs/source/guides/performance.rst]: Link to the TDS blog or the Feature Labs engineering blog?

Comment (kmax12) [reply docs/source/guides/performance.rst]: actually, let's do Feature Labs engineering blog

**Comment by kmax12** [comment]
Looks good. Merging

**Pull request merged**